internal/cmd/config/config.go

```diff
@@ -99,6 +100,15 @@ type Controller struct {
 	Description       string    `hcl:"description"`
 	Database          *Database `hcl:"database"`
 	PublicClusterAddr string    `hcl:"public_cluster_addr"`
+
+	// AuthTokenTimeToLive is the total valid lifetime of a token denoted by time.Duration
+	AuthTokenTimeToLive         string `hcl:"auth_token_time_to_live"`
```

Comment:
This shouldn't be a string, it should be an `interface{}` and use ParseDuration, like we do elsewhere.